apps/www/_blog/2023-07-25-supabase-lw8-hackathon.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Supabase Launch Week 8 Hackathon'
+description: Build an Open Source Project over 10 days. 5 prize categories.
+author: ant_wilson
+image: lw8-hackathon/thumbnail.png
+thumb: lw8-hackathon/thumbnail.png
+launchweek: 8
+tags:
+  - launch-week
+  - hackathon
+date: '2023-07-25'
+toc_depth: 2
+---
+
+[Supabase Launch Week 8](https://supabase.com/launch-week) is about to start and we're running an async hackathon alongside it.
+
+The hackathon starts Friday 4th August at 09:00 am PT and ends Sunday 13th August at 11:59 pm PT. You can win extremely limited edition Launch Week swag and add your name to the Supabase Hackathon Hall of Fame.
+
+For some inspiration check out all the [winners from previous hackathons](https://supabase.com/blog/tags/hackathon).
+
+This is the perfect excuse to "Build in a weekend, scale to millions", since you retain all the rights to your submissions, you can use the hackathon as a launch pad for your new Startup ideas, side-project, or indie hack.
```

Comment:
```suggestion
This is the perfect excuse to "Build in a weekend, scale to millions". Since you retain all the rights to your submissions, you can use the hackathon as a launch pad for your new Startup ideas, side-project, or indie hack.
```